code/modules/clothing/suits/ego_gear/aleph.dm

```diff
@@ -91,7 +91,7 @@
 	desc = "It is not as unpleasant to wear as it is to look at. \
 	In fact, it seems to give you an illusion of comfort and bravery."
 	icon_state = "adoration"
-	armor = list(RED_DAMAGE = 70, WHITE_DAMAGE = 40, BLACK_DAMAGE = 70, PALE_DAMAGE = 50)
+	armor = list(RED_DAMAGE = 70, WHITE_DAMAGE = 40, BLACK_DAMAGE = 80, PALE_DAMAGE = 50)
```

Comment:
I'd rather leave it as is. The black-specific armor should be MoSB.

Reply:
Ill put the 10 extra points into white instead to round it off